### PR TITLE
Update dockerfile

### PR DESCRIPTION
**Pull request by simonsso**

### CODE REVIEW

**Comment by MittWillson** [comment]
How about using `curl -s https://api.github.com/repos/Polkadex-Substrate/Polkadex/releases/latest | jq -r '.tag_name'` get latest release tag and using `--depth=1` for more less code clone

**Comment by simonsso** [comment]
> How about using `curl -s https://api.github.com/repos/Polkadex-Substrate/Polkadex/releases/latest | jq -r '.tag_name'` get latest release tag and using `--depth=1` for more less histories clone

I considered it, but there will be release tags for testnet releases too.

File: dockerfile [hunk 1]
```diff
@@ -1,20 +1,16 @@
 FROM ubuntu as builder
 RUN apt-get update
 RUN apt-get install -y git && apt-get install -y curl
-RUN git clone https://github.com/Polkadex-Substrate/Polkadex
+RUN git clone https://github.com/Polkadex-Substrate/Polkadex -b main-net-runtime
 RUN cd Polkadex && \
-    git checkout $(git describe --tags --abbrev=0) && \ # Gets the latest tag
-    curl -O -L https://github.com/Polkadex-Substrate/Polkadex/releases/download/$(git describe --tags --abbrev=0)/customSpecRaw.json && \ # Gets the latest Spec
```
Comment (arrudagates) [inline dockerfile:7]: Where are you getting this file if this line is removed?

Comment (arrudagates) [reply dockerfile:7]: I see, it's already on the repo.

**Review by arrudagates — APPROVED** [review]

**Pull request merged**